src/_shared/hooks/useWindowSize.ts

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export function useWindowSize() {
+  const isWindow = typeof window !== "undefined";
+  const [windowSize, setWindowSize] = React.useState({
```

Comment:
I think this was copied from somewhere <3 we never use React. to use state, effects etc. Moreover, why window should be undefined?

Reply:
Yeah, I copied it from StackOverflow ahah. The undefined was to prevent errors if the app was server rendered. In this case, it is definitely not necessary